.github/workflows/R-CMD-check-full.yaml

```diff
@@ -78,3 +78,13 @@ jobs:
         with:
           name: ${{ runner.os }}-r${{ matrix.config.r }}-results
           path: check
+
+      - name: Install codecov()
+        if: runner.os == 'macOS'
+        run: remotes::install_cran("covr")
+        shell: Rscript {0}
+
+      - name: Test coverage
+        if: runner.os == 'macOS'
+        run: covr::codecov()
+        shell: Rscript {0}
```

Comment:
these changes appear not to be in main

which is strange because i merged from main

i assume these are lines are outdated, and ill remove with confirmation

Reply:
these lines shouldn't be in the test full-test, they should just be in the single test